web-client/src/presenter/computeds/formattedMessages.test.js

```diff
@@ -146,4 +150,342 @@ describe('formattedMessages', () => {
       messages: [],
     });
   });
+
+  it('the filter dropdown values should be set correctly for the messages data', () => {
+    const result = runCompute(formattedMessages, {
+      state: {
+        messageBoxToDisplay: {
+          box: 'outbox',
+        },
+        messages: [
+          {
```

Comment:
💭  With all these messages in tests, maybe it'd be worth adding a `mockMessages` file like we have with `mockCase.js`